assignments/objects.js

```diff
@@ -11,54 +11,120 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: 'Example',
+  email: 'examples@you.edu',
+  gender: 'F'
+};
 
 // Write your intern objects here:
 
-
-// ==== Challenge 2: Reading Object Data ==== 
+const mitzi = {
+  id: 1,
+  name: 'Mitzi',
+  email: 'mmelloy0@psu.edu',
+  gender: 'F'
+};
+
+const kennan = {
+  id: 2,
+  name: 'Kennan',
+  email: 'kdiben1@tinypic.com',
+  gender: 'M',
+  speak: function() {
+    return `hello my name is ${this.name}`;
+  }
+};
+
+const keven = {
+  id: 3,
+  name: 'Keven',
+  email: 'kmummery2@wikimedia.org',
+  gender: 'M'
+};
+
+const gannie = {
+  id: 4,
+  name: 'Gannie',
+  email: 'gmartinson3@illinois.edu',
+  gender: 'F'
+};
+
+const antonietta = {
+  id: 5,
+  name: 'Antonietta',
+  email: 'adaine5@samsung.com',
+  gender: 'F',
+  multiplyNums: function(num1, num2) {
+    return num1 * num2;
+  }
+};
+
+// ==== Challenge 2: Reading Object Data ====
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
-
+console.log(mitzi['name']);
 // Kennan's ID
-
+console.log(kennan['id']);
 // Keven's email
-
+console.log(keven['email']);
 // Gannie's name
-
+console.log(gannie['name']);
 // Antonietta's Gender
-
-// ==== Challenge 3: Object Methods ==== 
+console.log(antonietta['gender']);
+// ==== Challenge 3: Object Methods ====
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+console.log(kennan.speak());
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+console.log(antonietta.multiplyNums(3, 4));
 
 // === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 
-// ==== Stretch Challenge: Nested Objects and the this keyword ==== 
+// ==== Stretch Challenge: Nested Objects and the this keyword ====
 
 // 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
 // 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  name: 'Susan',
+  age: 70,
+  speak: function() {
+    return `My name is ${this.name}`;
+  },
+  child: {
+    name: 'George',
+    age: 50,
+    speak: function() {
+      return `My name is ${this.name}`;
+    },
+    grandchild: {
+      name: 'Sam',
+      age: 30,
+      speak: function() {
+        return `My name is ${this.name}`;
+      }
+    }
+  }
+};
```

Comment:
how about this pattern, since we're getting ahead of ourselves with `this` :) 

```
function speak() {
     return `My name is ${this.name}`;
}
const parent = {
  name: 'Susan',
  age: 70,
  speak: speak,
  child: {
    name: 'George',
    age: 50,
    speak: speak,
    grandchild: {
      name: 'Sam',
      age: 30,
      speak: speak
    }
  }
};
```